### PR TITLE
add cheatsheet for tidyverse

### PR DESCRIPTION
**Pull request by Huimin-alice**
add a contributing file and a picture.

### CODE REVIEW

File: tidyverse_contribution.Rmd [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#cheatsheet for tidyverse
```
Comment (RandomWalker300) [inline tidyverse_contribution.Rmd]: 1. line 1: add a space after hashtag
2. line6: change your .png path to "resources/tidyverse_tutorial/tidyverseFinal.png"
3. remove .DS_Store, and resources/.DS_Store

**Review by RandomWalker300 — CHANGES_REQUESTED** [review]
line 1: please add a space after hashtag
line6: please change your .png path to "resources/tidyverse_tutorial/tidyverseFinal.png"
please remove .DS_Store, and resources/.DS_Store

**Comment by Huimin-alice** [comment]
all the problem listed as fixed, please check.

File: tidyverse_contribution.Rmd (hunk 1 above)
Comment (Huimin-alice) [reply tidyverse_contribution.Rmd]: all problems listed are fixed, please check.

**Comment by skyetim** [comment]
🎉 Congrats! Your contribution has been merged!

**Pull request merged**